src/main/java/uk/gov/ons/ctp/response/casesvc/service/CaseService.java

```diff
@@ -167,6 +169,86 @@ public Case findCaseByIac(final String iac) throws CTPException {
     return cazes;
   }
 
+  /**
+   * Find the paginated cases for a partyId ordered by create date descending.
+   *
+   * @param partyId the partyId
+   * @param maxCasesPerSurvey the maximum number of cases per survey to return
+   * @return the cases for the partyId
+   */
+  public List<Case> findCasesByPartyIdLimitedPerSurvey(
+      final UUID partyId, boolean iac, int maxCasesPerSurvey) {
+    log.info("Entering findCasesByPartyIdLimitedPerSurvey");
+
+    List<Case> cazes =
```

Comment:
🎨 I would just call this `cases` - we tend to only use these weird spellings when it conflicts with a keyword (like `clazz`)

Reply:
yes please!  Java using weird spellings for words was never my favourite practice >_<

Reply:
Still needs looking at